src/basic/strv.c

```diff
@@ -861,6 +861,24 @@ int strv_compare(char * const *a, char * const *b) {
         return 0;
 }
 
+bool strv_equal_ignore_order(char **a, char **b) {
+
+        /* Just like strv_equal(), but doesn't care about the order of elements or about redundant entries. */
```

Comment:
Maybe better to comment that difference in number of duplicated elements is also ignored.
E.g. `"a", "a", "b"` vs `"a", "b", "b"` vs `"a", "b"`: all of them are handled as equal.
Please add the test cases for such.